src/careers/current-openings/_single.html

```diff
@@ -96,23 +107,24 @@ <h2 class="u-mb0">Interested in Applying?</h2>
 
             <ul class="list list__links">
                 <li class="list_item">
-                    <a class="jump-link" href="#">
+                    <a class="jump-link" href="/working-at-cfpb/">
```

Comment:
Should this be `/careers/working-at-cfpb/` or `../working-at-cfpb`? (Prefer absolute URL.)

Here and for L:115